examples/with-tailwindcss/src/pages/index.astro

```diff
@@ -13,7 +11,8 @@ import styles from '../styles/global.css';
 		<meta charset="UTF-8" />
 		<link rel="icon" type="image/x-icon" href="/favicon.ico" />
 		<title>Astro + TailwindCSS</title>
-		<style set:html={styles}></style>
+		<!-- Imports the global styles -->
+		<link rel="stylesheet" href="/src/styles/global.css" />
```

Comment:
Definitely like this better! We're sure that this works with `--experimental-static-build`, though? IIRC some users reported that this approach only worked in `dev`.

Reply:
It doesn't work in `static build`!

My go-to approach in static build right now is actually using `import "../styles/global.css"` in the frontmatter, which adds it to the asset graph and automatically includes the `<link rel="stylesheet">` tag

My thought was we'd update examples with that syntax when we flip the switch and use static build by default, but maybe a comment in the `with-tailwindcss` example for now in case anyone uses it with the build flag?